src/app-layer-parser.h

```diff
@@ -103,6 +103,12 @@ void AppLayerParserThreadCtxFree(AppLayerParserThreadCtx *tctx);
 int AppLayerParserConfParserEnabled(const char *ipproto,
                                     const char *alproto_name);
 
+/** \brief Prototype for parsing functions */
+typedef int (*ParserFPtr)(Flow *f, void *protocol_state,
```

Comment:
As this is now public I like it to be prefixed with AppLayer.

Reply:
Should I use something like `AppLayerParserFPtr` ?
note: there is `ProbingParserFPtr` in `src/app-layer-detect-proto.h`

Reply:
Changed to `AppLayerParserFPtr`